lib/core/constants/dashboard_constants.dart

```diff
@@ -0,0 +1,8 @@
+class DashboardConstants {
```

Comment:
Mostly, classes like this are global and should be abstract, there's no need to instantiate this class.